### PR TITLE
Remove repositories configuration

### LINKED ISSUE #17: CA certificate issue while running conversion_host_enable/disable.yml with OSP
**conversion_host_enable.yaml issue**
I ran playbook for OSP conversion appliance with following sample inventory
```
all:
  hosts:
    conversion_host:
      v2v_host_type: openstack
      v2v_transport_methods:
        - vddk
      manageiq_provider_name: "osp"
  vars:
    v2v_vddk_package_name: "VMware-vix-disklib-6.7.1-10362358.x86_64.tar.gz"
    v2v_vddk_package_url: "http://hostname.com/VMware-vix-disklib-6.7.1-10362358.x86_64.tar.gz"
    manageiq_url: https://10.1.2.3
    manageiq_username: admin
    manageiq_password: smartvm
    manageiq_zone_id: 1
    manageiq_providers:
      - name: "osp"
        connection_configurations:
          - endpoint:
              role: default
              verify_ssl: false
```

Playbook fails with error 
```
TASK [oVirt.v2v-conversion-host : set_fact] *************************************************************************************************************************************************************************
ok: [localhost]

TASK [oVirt.v2v-conversion-host : set_fact] *************************************************************************************************************************************************************************
ok: [localhost]

TASK [oVirt.v2v-conversion-host : uri] ******************************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failed to validate the SSL certificate for 10.1.2.3:443. Make sure your managed systems have a valid CA certificate installed. You can use validate_certs=False if you do not need to confirm the servers identity but this is unsafe and not recommended. Paths checked for this platform: /etc/ssl/certs, /etc/pki/ca-trust/extracted/pem, /etc/pki/tls/certs, /usr/share/ca-certificates/cacert.org, /etc/ansible. The exception msg was: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:618)."}
	to retry, use: --limit @/usr/share/ovirt-ansible-v2v-conversion-host/playbooks/conversion_host_enable.retry

PLAY RECAP **********************************************************************************************************************************************************************************************************
localhost                  : ok=34   changed=5    unreachable=0    failed=1 
```

This do have workaround. Editing Line 9 of `/usr/share/ansible/roles/oVirt.v2v-conversion-host/tasks/install-manageiq-conversion-host.yml`, from `validate_certs: true` to `validate_certs: false` should run playbook successfully.

**conversion_host_disable.yaml issue**
Ran disable playbook with mentioned inventory, got following error - 
```

TASK [oVirt.v2v-conversion-host : assert] ***************************************************************************************************************************************************************************
ok: [new-qeconversion1] => {
    "changed": false, 
    "msg": "All assertions passed"
}

TASK [oVirt.v2v-conversion-host : include_tasks] ********************************************************************************************************************************************************************
included: /usr/share/ansible/roles/oVirt.v2v-conversion-host/tasks/uninstall.yml for new-qeconversion1

TASK [oVirt.v2v-conversion-host : set_fact] *************************************************************************************************************************************************************************
ok: [new-qeconversion1] => (item={u'connection_configurations': [{u'endpoint': {u'verify_ssl': False, u'role': u'default'}}], u'name': u'osp'})

TASK [oVirt.v2v-conversion-host : include_tasks] ********************************************************************************************************************************************************************
included: /usr/share/ansible/roles/oVirt.v2v-conversion-host/tasks/uninstall-provider-ssl-ca-chain.yml for new-qeconversion1

TASK [oVirt.v2v-conversion-host : Delete CA chain file in anchors] **************************************************************************************************************************************************
fatal: [new-qeconversion1]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'hostname'\n\nThe error appears to have been in '/usr/share/ansible/roles/oVirt.v2v-conversion-host/tasks/uninstall-provider-ssl-ca-chain.yml': line 2, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n---\n- name: Delete CA chain file in anchors\n  ^ here\n"}
	to retry, use: --limit @/usr/share/ovirt-ansible-v2v-conversion-host/playbooks/conversion_host_disable.retry

PLAY RECAP **********************************************************************************************************************************************************************************************************
new-qeconversion1          : ok=6    changed=0    unreachable=0    failed=1   
```

Workaround for disable playbooks - 
In `/usr/share/ansible/roles/oVirt.v2v-conversion-host/tasks/uninstall.yml` we need to comment Line 11, 
`include_tasks: "uninstall-provider-ssl-ca-chain.yml"`.

### PR DESCRIPTION
**Pull request by ghost**
Initially, we added some variables to allow configuration of additional repositories on the conversion hosts. It was mostly meant to expose SRPMS, but was never documented and the more I think of it, the more I feel that it doesn't belong to the conversion host role: enabling internal repositories should be done by another playbook/role that matches customer's environment. Otherwise, all roles should have this kind of variables...

This PR removes the configuration of local repositories through `v2v_repo_*` variables.
It also fixes the certificate check for ManageIQ.

Fixes https://github.com/oVirt/ovirt-ansible-v2v-conversion-host/issues/17

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: tasks/install-manageiq-conversion-host.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@
     user: "{{ manageiq_username }}"
     password: "{{ manageiq_password }}"
     method: "POST"
-    validate_certs: true
+    validate_certs: false
```
Comment (nyoxi) [inline tasks/install-manageiq-conversion-host.yml]: I'd rather see this as configuration option that defaults to "true".

Comment (ghost) [reply tasks/install-manageiq-conversion-host.yml]: Good point. It should be better now.

**Comment by nyoxi** [comment]
ci test please

**Pull request merged**